oak_runtime/introspection_browser_client/build

```diff
@@ -2,8 +2,12 @@
 #
 # Build browser introspection client.
 
-# Install npm modules
-npm ci
+# Install npm modules if needed
+readonly CURRENT_PACKAGE_LOCK="./package-lock.json"
+readonly LAST_INSTALLED_PACKAGE_LOCK="./node_modules/.cache/last-install-package-lock.json"
+if ! cmp -s "$CURRENT_PACKAGE_LOCK" "$LAST_INSTALLED_PACKAGE_LOCK"; then
```

Comment:
Would this read better with `diff` instead of `cmp`? (I must admit I had never heard of `cmp` before)

Reply:
Could be! `cmp` is faster since it stops at the first byte difference, but admittedly with files this small it probably wouldn't make a practical difference. 

Do you think I should switch to `diff` for legibility?